<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_CtrUser_Admin</class>
 <widget class="QWidget" name="Widget_CtrUser_Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 255, 230);</string>
  </property>
  <widget class="QTableWidget" name="tbl_Users">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>631</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Nombres</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cumplea√±os</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_buscar">
      <property name="text">
       <string>buscar@gmail.com</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Orden</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_orden">
      <item>
       <property name="text">
        <string>Preorden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inorden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Postorden</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Aplicar">
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
